src/ui_fsmenu/addons.cc

```diff
@@ -416,6 +416,11 @@ AddOnsCtrl::AddOnsCtrl(MainMenu& fsmm, UI::UniqueWindow::Registry& reg)
 		refresh_remotes();
 		tabs_.activate(1);
 	});
+	tabs_.sigclicked.connect([this]() {
+		if (remotes_.empty() && tabs_.active() == 1) {
```

Comment:
```suggestion
		if (tabs_.active() == 1 && remotes_.size() <= 1) {
```
to retry more often if a network error occurred previously